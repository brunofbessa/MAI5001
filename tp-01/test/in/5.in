O
3
p3
3.33
O
1
p1
1.11
O
2
p2
2.22
O
1
px
9.00
L
F
