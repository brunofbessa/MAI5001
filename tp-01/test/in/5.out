Ok
Ok
Ok
Duplicado
1 - p1 - 1.11
2 - p2 - 2.22
3 - p3 - 3.33
Fim
